tests/unit/test_assemble.py

```diff
@@ -180,14 +180,14 @@ def setUp(self):
             self.seq = [i.strip() for i in fh.readlines()]
 
     def test_deterministic_assembly(self):
-        for i in range(10):
+        contig_lengths = set()
+        for i in range(20):
             random.shuffle(self.seq)
             contigs = assemble(self.seq,
                                111,
                                min_edge_trim_weight=3,
                                assembly_max_paths=8,
                                assembly_min_uniq=0.1,
                                min_complexity=0.1)
-            self.assertEqual(1, len(contigs))
-            self.assertEqual(387, contigs[0].score)
-            self.assertEqual('AAGGGAAAGGAAAGGAAAGGAAAGAAAGAAAGAAAGAAAGAAAGAAAGAAAGAAAGGAAAGGAAGACCTACAGCACACACAAAAAGGGGTTCAAAATGGATTAATAACTTAAATGTAAGAGCTTAAACTATAAAACTCCTTGACAAAGCATAGAGGGAAACTACTTGTAT', contigs[0].seq)
+            contig_lengths.add(len(contigs))
+            self.assertEqual(1, len(contig_lengths))
```

Comment:
Should we maybe add contigs[0].seq to the set we are checking instead of just the length? otherwise we might have 2 different sequences of the same length

Reply:
Good idea!